release.py

```diff
@@ -175,6 +179,27 @@ def make_hotfix_branch(github_repository, release_set, release_version, release_
         release_version=release_version
     )
 
+def prepare_token_remote(api_client, github_user):
+    try:
+        print('Checking if a token remote exists...')
+        command = 'git remote -v | grep {remote_name}'.format(remote_name=remote_name)
```

Comment:
```suggestion
        command = f'git remote -v | grep {remote_name}'
```

If all are already using Python 3.6, then this and other strings can be simplified